example/test/avoid_unnecessary_setstate_test.dart

```diff
@@ -0,0 +1,78 @@
+// ignore_for_file: avoid_global_state
+
+/// Check unnecessary setstate fail
+/// `avoid_unnecessary_setstate`
+import 'package:flutter/material.dart';
+
+class MyWidget extends StatefulWidget {
```

Comment:
If you are using any code from the DCM package, please, add a license at the very beginning of the file.